src/content/web3/index.md

```diff
@@ -0,0 +1,148 @@
+
+_In Web2, you can refuse to use a platform; in Web3, you can vote to change it._
+
+<ButtonLink to="/dao/">
+  More on DAOs
+</ButtonLink>
+
+#### Censorship resistance {#censorship-resistance}
+
+The power dynamic between platforms and content creators is massively imbalanced.
+
+OnlyFans is a user-generated adult content site with over 1-million content creators, many of which use the platform as their primary source of income. In August 2021, OnlyFans announced plans to ban sexually explicit content. The announcement sparked outrage amongst creators on the platform, who felt they were getting robbed of an income on a platform they helped create. After the backlash, the decision got quickly reversed.
+
+Web 2.0 requires content creators to trust platforms not to change the rules, but censorship resistance is a native feature of a web3 platform.
+
+### Identity {#identity}
+
+Traditionally, you would create an account for every platform you use. For example, you might have a Twitter account, a YouTube account, and a Reddit account. Want to change your display name or profile picture? You have to do it across every account. You can use social sign-ins in some cases, but this presents a familiar problem—censorship. In a single click, these platforms can lock you out of your entire online life. Even worse, many platforms require you to trust them with personally identifiable information to create an account.
+
+Web3 solves these problems by allowing you to control your digital identity with an Ethereum address and ENS profile. Using an Ethereum address provides a single login across platforms that is secure, censorship-resistant, and anonymous.
+
+<ButtonLink to="https://login.xyz/">
+  Sign-in with Ethereum
+</ButtonLink>
+
+### Native payments {#native-payments}
+
+Web2's payment infrastructure relies on banks and payment processors, excluding people without bank accounts or those who happen to live within the borders of the wrong country.
+Web3 uses tokens like [ETH](/eth/) to send money directly in the browser and requires no trusted third party.
+
+<ButtonLink to="/eth/">
+  More on ETH
+</ButtonLink>
+
+## Web3 limitations {#web3-limitations}
+
+Despite the numerous benefits of Web3 in its current form, there are still many limitations that the ecosystem must address for it to flourish.
+
+### Accessibility {#accessibility}
+
+Web3 is less likely to be utilized in less-wealthy, developing nations due to high transaction fees. On Ethereum, these challenges are being solved through [network upgrades](/upgrades/) and [layer 2 scaling solutions](/developers/docs/scaling/). The technology is ready, but we need higher levels of adoption on layer 2 to make Web3 accessible to everyone.
```

Comment:
A subset of web3 features can be done prior to solving the scalability problem.  For example, sign-in with Ethereum doesn't require any money to utilize.  Also, any application builder can choose *today* to build their data platform as a layer 2, there is no need to wait for someone else to do it.

I think this section in general should remain so that we are honest about the reality on the ground, but I think there is value in highlighting that stuff can be built *today* that follows web3 principals without needing to wait for others to solve hard problems first.